packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -322,8 +322,114 @@ export default {
 
 @import '../kotti-style/_variables.scss';
 
+table.kt-table {
```

Comment:
for other reviewers:
moved from here

https://github.com/3YOURMIND/kotti/pull/787/files#diff-604842dfb9282c755ebaa023a8363c33fdb4c010b18bff8f8170c49e41acf633L3